"container_label_annotation_kubernetes_io/config_seen" is not a valid label name for metric "container_last_seen"

Am I doing something wrong? Using Windows Server 1809

```
C:\k>docker run --rm -p 9030:9030  -v \\.\pipe\docker_engine:\\.\pipe\docker_engine alexvaut/cstatsexporter:latest
Starting...
Configuration read. Scrap time = 15 seconds.
6d7cdb381adb alexvaut/cstatsexporter:latest
07d6bbcd6be2 kubeletwin/pause
3c76d9a4c929 mcr.microsoft.com/windows/servercore@sha256:2ecf1e2987b91b41f576afd7f56aa40c9ddbc691d7b6b066c64d8f27fb3070ca
0434b0556fb2 mcr.microsoft.com/windows/servercore@sha256:2ecf1e2987b91b41f576afd7f56aa40c9ddbc691d7b6b066c64d8f27fb3070ca
a4ef74b2ac55 kubeletwin/pause
299ba34f6466 kubeletwin/pause
panic: descriptor Desc{fqName: "container_last_seen", help: "Last time a container was seen by the exporter.", constLabels: {}, variableLabels: [container_label_annotation_kubernetes_io/config_seen container_label_annotation_io_kubernetes_container_terminationMessagePolicy container_label_app image name container_label_annotation_io_kubernetes_container_restartCount container_label_annotation_io_kubernetes_container_hash container_label_io_kubernetes_pod_name container_label_annotation_io_kubernetes_container_terminationMessagePath container_label_annotation_io_kubernetes_pod_terminationGracePeriod container_label_io_kubernetes_pod_namespace container_label_controller-revision-hash container_label_component container_label_pod-template-generation id container_label_io_kubernetes_pod_uid container_label_pod-template-hash container_label_release container_label_io_kubernetes_container_logpath container_label_io_kubernetes_sandbox_id container_label_annotation_kubernetes_io/config_source container_label_io_kubernetes_container_name container_label_io_kubernetes_docker_type]} is invalid: "container_label_annotation_kubernetes_io/config_seen" is not a valid label name for metric "container_last_seen"

goroutine 1 [running]:
github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc0002ac1e0, 0xc0003574d0, 0x1, 0x1)
        C:/Users/avautier/go/pkg/mod/github.com/prometheus/client_golang@v1.1.0/prometheus/registry.go:399 +0xb4
github.com/prometheus/client_golang/prometheus.MustRegister(...)
        C:/Users/avautier/go/pkg/mod/github.com/prometheus/client_golang@v1.1.0/prometheus/registry.go:176
github.com/prometheus/client_golang/prometheus/promauto.NewGaugeVec(0x0, 0x0, 0x0, 0x0, 0xae877d, 0x13, 0xafbcd0, 0x2f, 0x0, 0xc0000f0e00, ...)
        C:/Users/avautier/go/pkg/mod/github.com/prometheus/client_golang@v1.1.0/prometheus/promauto/auto.go:174 +0xff
main.main()
        C:/projects/cStatsExporter/main.go:153 +0x5f5
```

--- a/config.yml
+++ b/config.yml
@@ -1,2 +1,3 @@
 scrapeIntervalSeconds: 15
 port: 9030
+kubernetes: false

--- a/config/config.go
+++ b/config/config.go
@@ -4,4 +4,5 @@ package config
 type Configurations struct {
 	ScrapeIntervalSeconds int64
 	Port                  int
+	Kubernetes            bool			 
 }

--- a/main.go
+++ b/main.go
@@ -125,7 +125,6 @@ func main() {
 	for _, info := range infos {
 		labels := BuildLabels(info.ID, false)
 		for labelName, _ := range labels {
-			log.Println(labelName)
 			labelNamesM[labelName] = ""
 		}
 	}
@@ -408,8 +407,9 @@ func BuildLabels(id string, filter bool) prometheus.Labels {
 		labels = prometheus.Labels{"id": "/docker/" + id, "image": info.Config.Image, 
 		    "pod": info.Config.Labels["io.kubernetes.pod.name"], 
 		    "namespace": info.Config.Labels["io.kubernetes.pod.namespace"],
-		    "beta_kubernetes_io_os": "windows",
-		    "kubernetes_io_role": "node"}
+			"node_name": os.Getenv("NODE_NAME"),
+			"beta_kubernetes_io_os": "windows",
+			"kubernetes_io_role": "node"}
 	} else {
 		labels = prometheus.Labels{"id": "/docker/" + id, "image": info.Config.Image, "name": info.Name}
 		for labelName, labelValue := range info.Config.Labels {

--- a/main.go
+++ b/main.go
@@ -63,6 +63,7 @@ var (
 	hostLabelNamesM               = prometheus.Labels{NodeIdLabel: "", HostnameLabel: "", NodeNameLabel: ""}
 	labelNamesM                   = prometheus.Labels{}
 	metrics                       = make([]interface{}, 0)
+	config c.Configurations 	  = GetConfig()
 )
 
 func WaitForCtrlC() {
@@ -115,7 +116,6 @@ func main() {
 	//}()
 
 	fmt.Println("Starting...")
-	var config c.Configurations = GetConfig()
 	fmt.Printf("Configuration read. Scrap time = %d seconds.\n", config.ScrapeIntervalSeconds)
 
 	GatherMetrics()
@@ -125,6 +125,7 @@ func main() {
 	for _, info := range infos {
 		labels := BuildLabels(info.ID, false)
 		for labelName, _ := range labels {
+			log.Println(labelName)
 			labelNamesM[labelName] = ""
 		}
 	}
@@ -402,28 +403,35 @@ func DeleteMetrics(labels prometheus.Labels) {
 
 func BuildLabels(id string, filter bool) prometheus.Labels {
 	info := infos[id]
-	var labels = prometheus.Labels{"id": "/docker/" + id, "image": info.Config.Image, "name": info.Name}
-
-	for labelName, labelValue := range info.Config.Labels {
-		labels[NormalizeLabel(labelName)] = labelValue
-	}
-
-	if filter {
-		//add labels that are missing (according to labelNamesM)
-		for k, _ := range labelNamesM {
-			if _, ok := labels[k]; !ok {
-				labels[k] = ""
-			}
+	var labels = prometheus.Labels{}
+	if config.Kubernetes {
+		labels = prometheus.Labels{"id": "/docker/" + id, "image": info.Config.Image, 
+		    "pod": info.Config.Labels["io.kubernetes.pod.name"], 
+		    "namespace": info.Config.Labels["io.kubernetes.pod.namespace"],
+		    "beta_kubernetes_io_os": "windows",
+		    "kubernetes_io_role": "node"}
+	} else {
+		labels = prometheus.Labels{"id": "/docker/" + id, "image": info.Config.Image, "name": info.Name}
+		for labelName, labelValue := range info.Config.Labels {
+			labels[NormalizeLabel(labelName)] = labelValue
 		}
 
-		//remove labels that are unknown (according to labelNamesM)
-		for k, _ := range labels {
-			if _, ok := labelNamesM[k]; !ok {
-				delete(labels, k)
+		if filter {
+			//add labels that are missing (according to labelNamesM)
+			for k, _ := range labelNamesM {
+				if _, ok := labels[k]; !ok {
+					labels[k] = ""
+				}
+			}
+
+			//remove labels that are unknown (according to labelNamesM)
+			for k, _ := range labels {
+				if _, ok := labelNamesM[k]; !ok {
+					delete(labels, k)
+				}
 			}
 		}
 	}
-
 	return labels
 }
 

--- a/main.go
+++ b/main.go
@@ -407,7 +407,7 @@ func BuildLabels(id string, filter bool) prometheus.Labels {
 		labels = prometheus.Labels{"id": "/docker/" + id, "image": info.Config.Image, 
 		    "pod": info.Config.Labels["io.kubernetes.pod.name"], 
 		    "namespace": info.Config.Labels["io.kubernetes.pod.namespace"],
-			"node_name": os.Getenv("NODE_NAME"),
+			"kubernetes_io_hostname": os.Getenv("NODE_NAME"),
 			"beta_kubernetes_io_os": "windows",
 			"kubernetes_io_role": "node"}
 	} else {

